NPC regular state correction

sky1683589933 opened the pull request:
NPC regular state correction

sky1683589933 committed:
NPC regular state correction

NPC regular state correction

neon-dev reviewed (changes requested):
Non-default states are usually only relevant for one spawn position, so it would be better to set them in the spawn templates. Geolus for example, can be set like so in `spawns/Npcs/210030000_Verteron.xml`:
```xml
<!-- Geolus -->
<spawn npc_id="203149" respawn_time="295">
	<spot x="1676.37" y="2270.09" z="166.773" h="35" state="85"/>
</spawn>
```
Please revert all changes in the npc templates and set the states in the individual spawn templates.
Side note: Committing huge files like npc_templates.xml bloats the repository over time, so they should only be changed if necessary.

sky1683589933 commented:
Yeah, submitting a huge template to fix a few irrelevant NPC states is a foolish approach!
Just add it to the spawns, how could I make such a mistake...

sky1683589933 committed:
Add a reject task option for Quest 2122

Add dialog reject  option to avoid unresponsive clicking the reject button in the client dialog.

neon-dev committed:
Revert "NPC regular state correction"

This reverts commit 02ea78d008749e56448c6fd032b6e68b116dd6d7.

neon-dev committed:
Set states in spawn templates
